appstudio-controller/controllers/appstudio.redhat.com/environment_controller.go

```diff
@@ -320,61 +363,57 @@ func generateDesiredResource(ctx context.Context, env appstudioshared.Environmen
 					"DeploymentTargetClaim not found while generating the desired Environment resource", &env,
 					EnvironmentConditionErrorOccurred, metav1.ConditionTrue, EnvironmentReasonErrorOccurred, log); err != nil {
 
-					return nil, true, fmt.Errorf("unable to update environment status condition. %v", err)
+					return nil, errorConditionSet_false, fmt.Errorf("unable to update environment status condition. %v", err)
```

Comment:
In many cases we were returning an error, and returning true: I switched all these cases to return false, instead.